docs/index.html

```diff
@@ -0,0 +1,96 @@
+        <img src="image/cat_image.jpg" alt="catimage.com" width="250" height="250">
+        <img src="image/Cat NASA201.jpg" alt="catimage2.com" width="250" height="250">
+        <img src="image/Cat NASA201.12.jpg" alt="catimage3.com" width="250" height="250">
+        <img src="image/Cat NASA201.13.jpg" alt="catimage3.com" width="250" height="250">
+
+        
+    </main>
+<h2>Introduction</h2>
+<p>
+    Cats: Feline Majesty in Domestic Grace
+    
+    Cats, with their enigmatic allure and independent spirits, have long captivated the hearts of humans. Revered in ancient civilizations and cherished in contemporary homes, these graceful creatures embody a unique blend of elegance and mystique.
+    </p>
+    <h2> Let's Explore the Cat Kingdom</h2>
+    <p>One of the most striking features of cats is their innate sense of autonomy. Unlike many domesticated animals, cats maintain a distinct aura of independence, navigating their surroundings with an air of self-assurance. Their aloof demeanor often belies a deep-rooted intelligence, as evidenced by their ability to problem-solve and adapt to diverse environments.
+
+        Beyond their independent nature, cats possess an unparalleled grace that is both captivating and mesmerizing. From the fluidity of their movements to the sleek contours of their bodies, every aspect of a cat's form exudes an innate sense of beauty. Whether lounging in the sun or prowling through moonlit shadows, cats move with a sense of purpose that commands admiration and respect.</p>
+
+        
+          
+          
+        <!DOCTYPE html>
+<html>
+<style>
+table, th, td {
+  border:1px solid black;
+}
+
+th, td {
+  background-color: #96D4D4;
+}
+</style>
+<body>
+
+<h2>Here are some of the Names and where they come from:</h2>
+
+<table style="width:100%">
+            <tr>
+              <td>America</td>
+              <td>Europe</td>
+              <td>Age</td>
+            </tr>
+        </tr>
+        <tr>
+          <td>American Shorthair</td>
+          <td>German Rex</td>
+          <td>15</td>
+        </tr>
+        <tr>
+            <td>Bob Tail</td>
+            <td>Havana Brown</td>
+            <td>12</td>
+          </tr>
+          <tr>
+            <td>Cylon</td>
+            <td>Cornish Rex</td>
+            <td>15</td>
+          </tr>
+          </table>
+          </table>
+        
+
+          <h2>You can join the Cat's Community :</h2>
+
+          <form>
+            <label for="fname">First name:</label><br>
+            <input type="text" name="first_name"><br>
+            <label for="fname">Last name:</label><br>
+            <input type="text" name="last_name"><br>
+            <button type="submit"> Submit</button>
+          </form>
+
+          <h2>If you have some questions just click the link below:
+                
+          </h2>
+          <p><a href="mailto:23055331@saultcollege.com">Here is my email address you can click this link:</a></p>
```

Comment:
In order for the form to submit the form data automatically, you need to use the `mailto` URL as the `action` of the form, as in `<form action="mailto:..">`